website/data/users.yml

```diff
@@ -1,3 +1,7 @@
+- caption: Facebook
+  image: "/images/users/facebook-200x100.png"
+  infoLink: https://facebook.com
+  pinned: true
```

Comment:
I feel like we should randomize these on each deploy so that everyone gets their time in the spotlight.

Reply:
Or maybe randomize them on page load..

Reply:
#3868